plugin/mock_service_generator_test.go

```diff
@@ -0,0 +1,41 @@
+// Automatically generated by MockGen. DO NOT EDIT!
```

Comment:
Wondering why mockgen doesn't produce a `// Code generated by mockgen` comment so theses would be collapsed.